backend/templates/home-without_content.html

```diff
@@ -0,0 +1,117 @@
+{% extends 'base.html' %}
+{% load static %}
+{% load i18n %}
+{% block title %}
+    Home
+{% endblock %}
+
+{% block header %}
+
+    <meta name="description"
+          content="Der Glaube daran Mitmenschen in Not helfen können, die Liebe zum Fach Medizin und der Zusammenhalt untereinander sind in den letzten Wochen und Monaten im Gesundheitswesen noch viel präsenter denn je.">
```

Comment:
I agree that it is difficult to make these editable with the cms as it is not WYSIWYG. However, especially these will be edited by social media people, maybe even frequently when changing "campaign slogans" or stuff like that. Would that be possible?

Reply:
We would wanna use something like https://github.com/nephila/djangocms-page-meta or https://pypi.org/project/django-meta/. Feel free to open an issue for this